content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -0,0 +1,81 @@
+===================================================
+Use product variants on quotations and sales orders
+===================================================
+
+Product variants can be added to quotations and sales orders using the Product Configurator,
+Variant Grid Entry, or both.
+
+Configuration
+=============
+
+Before using product variants on your quotations and sales orders, you first need to activate one
+or both settings. To enable product variants on quotations and sales orders, head to the
```

Comment:
Deleting this sentence ("Before using product variants... both settings.") that was accidentally left in after the first edit
```suggestion
To enable product variants on quotations and sales orders, head to the
```